src/pages/functional-testing-framework/backward-incompatible-changes.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Backward incompatible changes | Commerce Testing
+description: Learn about major chnages to the Functional Testing Framework that may require specific action to ensure third-party tests continue working as expected with Adobe Commerce and Magento Open Source tests.
+---
+
+# MFTF 3.0.0 backward incompatible changes
+
+This page highlights backward incompatible changes between releases that have a major impact and require detailed explanation and special instructions to ensure third-party tests continue working with Magento core tests.
+
+## Minimum supported PHP version changes
+
+We changed the minimum PHP version requirement from 7.0 to 7.3. Because of the PHP version requirement change, this MFTF version only supports Magento 2.4 or later.
```

Comment:
```suggestion
We changed the minimum PHP version requirement to 7.4. Because of the PHP version requirement change, this MFTF version only supports Magento 2.4 or later.
```